### PR TITLE
UPD : added rest API for Student

### LINKED ISSUE #12: REST API for Students
* Refer to the course API for best practices and reference.

### PR DESCRIPTION
**Pull request by archit-1997**
Added rest API for Student

closes #12

### CODE REVIEW

**Review by rohan23chhabra — CHANGES_REQUESTED** [review]
Update server pom to use core version 1.0.2

**Comment by archit-1997** [comment]
> Update server pom to use core version 1.0.2

Updated !!

File: src/main/java/org/cms/server/student/StudentService.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+		return byId;
+	}
+
+	// Returns the id of the added student
+	public String addStudent(Student student) {
+		logger.info(String.format("[%s] addStudent HIT", addStudentPrefix));
+		logger.info(String.format("[%s] Student received to be added - %s", addStudentPrefix, student));
+
+		Student savedstudent = studentRepository.save(student);
```
Comment (rohan23chhabra) [inline src/main/java/org/cms/server/student/StudentService.java]: CamelCase please!

Comment (archit-1997) [reply src/main/java/org/cms/server/student/StudentService.java]: Done

**Review by rohan23chhabra — APPROVED** [review]

**Pull request merged**